### PR TITLE
Fix for unknown `cds.root`

### PR DESCRIPTION
**Pull request by IainSAP**
Fix for: https://github.com/SAP/open-ux-tools/issues/1562

Updates to `@sap-ux/project-access:getCapModelAndServices`

- Sets cds.root from project root
- Adds logger to log cds root, path and version found for debugging and support
- Add tests to cover new loc

### CODE REVIEW

**Review by devinea — COMMENTED** [review]
Code changes lg. 
Do we need some test to cover this flow?

File: packages/project-access/package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     },
     "dependencies": {
         "@sap-ux/ui5-config": "workspace:*",
+        "@sap-ux/logger": "workspace:*",
```
Comment (Klaus-Keller) [inline packages/project-access/package.json:37]: Shouldn't this be a `devDependency` as we use Logger as type only?

**Review by Klaus-Keller — COMMENTED** [review]
@IainSAP, changeset is missing and a small comment about dependency, otherwise looks good.

**Comment by IainSAP** [comment]
Wasnt supposed to be ready for review...not sure how that happened

**Review by devinea — APPROVED** [review]
re-approving. Thanks @IainSAP

**Review by Klaus-Keller — APPROVED** [review]
Thanks @IainSAP!
- Code to set `cds.root` and logging looks good
- Test coverage is great
- I've tested the solution to set `cds.root` locally

Approved from my side.

**Comment by IainSAP** [comment]
Thanks...

**Pull request merged**